internal/controllers/reconciliation/status_test.go

```diff
@@ -177,67 +179,3 @@ func TestReconcileStatus(t *testing.T) {
 }
 
 func isNotReady(state apiv1.ResourceState) bool { return state.Ready == nil }
-
-// TestStatusNegative covers a weird corner case where a resource fails a patch but is still marked as reconciled.
```

Comment:
This is no longer needed since the corner case it tested is just... not possible any more.